### PR TITLE
added note about exchange email location

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#965

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/dlp-microsoft-teams.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ To perform this task, you must be assigned a role that has permissions to edit D
 5. On the **Choose locations** tab, keep the default setting of all locations, or select **Let me choose specific locations**, and then choose **Next**.<br/>If you chose specific locations, select them for your DLP policy, and then choose **Next**.<br/>![DLP policy locations](media/dlp-teams-selectlocationsnewpolicy.png)<br/>
     > [!NOTE]
     > If you want to make sure documents that contain sensitive information are not shared inappropriately, make sure **SharePoint sites** and **OneDrive accounts** are turned on, along with **Teams chat and channel messages**.
+    > Channels in Microsoft Teams are stongly depend on Exchange Online functionality. Make sure that **Exchange email** location is also enabled for the policies that should be applied for the content of the channels.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-microsoft-teams.md]:
```suggestion
    > Channels in Microsoft Teams are strongly dependent on Exchange Online functionality. Make sure that **Exchange email** location is also enabled for the policies that should be applied for the content of the channels.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**